### DISCUSSION TITLE
`Live-Preview`: Improve window rendering with larger specified `breakpoints`.

### DISCUSSION THREAD

**Discussion by PatrikKozak** [payloadcms/payload#4588] [Feature Requests & Ideas]
Suggestion by @ericuldall in #4576 

Larger breakpoints tend to render off screen if larger than the available width of the window.

A few options to improving the behavior of this would be:

- Make the screen proportional based on the breakpoint size and width available, similar to how chrome does this in their mobile/table screen view (preferred approach. Something like this:
![image](https://github.com/payloadcms/payload/assets/35232443/d4735c33-f980-4a94-92bd-610ef11a38d1)
- Add in overflow scroll

**Comment by ericuldall** [comment]
![image](https://github.com/payloadcms/payload/assets/3223296/d0eaf089-1770-4bbf-8f8e-08e1e1b5b533)
The other problem is if I scale down to 50%, why does the image move all the way to the right side of the preview area? I think there's some other bugs here.

**Reply by ericuldall** [reply]
I do generally like the idea of thinking in terms of aspect ratio rather than exact size. It's a preview and I think fitting the correct aspect ratio as large as possible in the preview area is ideal.

**Comment by tijsomnicasa** [comment]
The old plugin for previews (pemedia/payload-visual-editor) used to handle this with transforming the div
